docs/querying/timeseriesquery.md

```diff
@@ -126,9 +126,10 @@ The grand totals row will appear as the last row in the result array, and will h
 row even if the query is run in "descending" mode. Post-aggregations in the grand totals row will be computed based
 upon the grand total aggregations.
 
-## Zero-filling
+## NULL filling
```

Comment:
I wonder if we should call this "Empty filling" or "Empty bucket filling" or something similar?

It doesn't actually always fill with zeros or nulls depending on the mode, rather it fills with the default values of the aggregator functions. Since they aggregate no rows, only the default value is there. https://github.com/apache/druid/blob/master/docs/querying/sql-aggregations.md has the default values of the aggregators.